src/components/Register.js

```diff
@@ -0,0 +1,105 @@
+import { useRef, useState } from "react";
+
+const Register = () => {
+  // const useRef = useRef();
+  // const errRef = useRef();
+
+  const [userDetails, setUserDetails] = useState({
+    fname: "",
+    lname: "",
```

Comment:
`firstName` and `lastName` are not too long

Reply:
okay sir